docs/parcool-guide-on-web-v3.0.0.0/actions/cling_to_cliff.md

```diff
@@ -2,7 +2,7 @@
 
 ---
 
-You can cling to cliffs. You can also climb up onto the cliffs by pressing jump key.
+You can cling to cliffs. You can also climb up onto the cliffs by pressing jump key or the bound key of your choice (unbound by default), or use the release cliff key to release it (unbound by default)
```

Comment:
This is a document for **v3.0.x.x**, outdated version. Description for latest changes should be written in `parcool-guide-on-web-latest`